### ISSUE TITLE
Evaluate widget markup

### ISSUE THREAD

**Issue by davidakennedy** [davidakennedy/accessible-zen#73]
For better accessibility, individual widgets should be wrapped in divs and the entire "sidebar" should be an `aside` element.
